iroha2-query-tool/README.md

```diff
@@ -0,0 +1,38 @@
+# Iroha2 query tool
+
+## Build 
+```shell
+docker build -t iroha2-query-tool .
+```
+Should be rebuild if script or config changed
+
+## Run
+```shell
+docker run --network="host" iroha2-query-tool "user:pass@example.com:port" last_tx  genesis@genesis
+```
+Get list of commands and options
+```shell
+docker run --network="host" iroha2-query-tool -h
+
+usage: iroha2-query-tool.py [-h] [--public_key PUBLIC_KEY]
+                            [--private_key PRIVATE_KEY]
+                            iroha_url {account_assets,asset_details,last_tx}
+                            ...
+
+Execute queries to iroha2
+
+positional arguments:
+  iroha_url             Url of iroha node (user:pass@example.com:port)
+  {account_assets,asset_details,last_tx}
+    account_assets      Find account assets by account id
+    asset_details       Find asset details by asset id
+    last_tx             Find last transactions by account id
```

Comment:
could you extend the list with `last block` as stated in the task description please

Reply:
Done, also made pretty printing of responses